src/main/java/uk/ac/ebi/eva/contigalias/dus/NCBIAssemblyReportReader.java

```diff
@@ -95,6 +97,99 @@ protected void parseAssemblyData(String line) {
     }
 
     protected void parseChromosomeLine(String[] columns) {
+        ChromosomeEntity chromosomeEntity = getChromosome(columns);
+        if (chromosomeEntity == null) {
+            return;
+        }
+
+        if (assemblyEntity == null) {
+            assemblyEntity = new AssemblyEntity();
+        }
+        chromosomeEntity.setAssembly(this.assemblyEntity);
+
+        List<ChromosomeEntity> chromosomes = this.assemblyEntity.getChromosomes();
+        if (chromosomes == null) {
+            chromosomes = new LinkedList<>();
+            assemblyEntity.setChromosomes(chromosomes);
+        }
+        chromosomes.add(chromosomeEntity);
+    }
+
+    protected void parseScaffoldLine(String[] columns) {
+        ChromosomeEntity scaffoldEntity = getScaffold(columns);
+        if (scaffoldEntity == null) {
+            return;
+        }
+
+        if (assemblyEntity == null) {
+            assemblyEntity = new AssemblyEntity();
+        }
+        scaffoldEntity.setAssembly(this.assemblyEntity);
+
+        List<ChromosomeEntity> scaffolds = this.assemblyEntity.getChromosomes();
+        if (scaffolds == null) {
+            scaffolds = new LinkedList<>();
+            assemblyEntity.setChromosomes(scaffolds);
+        }
+        scaffolds.add(scaffoldEntity);
+    }
+
+    public static AssemblyEntity getAssemblyEntity(List<String> lines) {
```

Comment:
Similar to the other comment, if this means the `parseAssemblyData` method isn't being used anymore, would prefer it be removed rather than have duplicate logic.

Reply:
removed

Reply:
I think it's [still there](https://github.com/nitin-ebi/contig-alias/blob/ingest_assembly/src/main/java/uk/ac/ebi/eva/contigalias/dus/NCBIAssemblyReportReader.java#L64), actually I'm not sure either of the report readers' parseReport methods are being used since they parse the whole report in one go rather than batching

Reply:
You are right, they are not used anymore except for tests.
I have deleted them and  updated the tests accordingly.